### ISSUE TITLE
[Bug] Perpetual prompting with yarn package manager

### ISSUE THREAD

**Issue by ilan-dahan** [webhintio/hint#5198]
# 🐞 Bug report

## Description
Using Yarn as a package manager and since latest update of webhint CLI, it is impossible to use it since messages keep asking which version of hint subpackages should be used. With team, we use bots (Renovate) to automatically update so everything is blocked with this behavior.
Thanks for your work team and please provide a workaround soon 😄 Say hi to Nellie for us!

Important notice: have used for a while without bug so it may be linked to latest versions.

## Details
Hint version: 7.0.0

CLI outputs: 
```zsh
warning @hint/hint-minified-js > @hint/utils-network > request@2.88.2: request has been deprecated, see https://github.com/request/request/issues/3142
warning @hint/hint-minified-js > @hint/utils-network > request > uuid@3.4.0: Please upgrade  to version 7 or higher.  Older versions may use Math.random() in certain circumstances, which is known to be problematic.  See https://v8.dev/blog/math-random for details.
warning @hint/hint-minified-js > @hint/utils-network > request > har-validator@5.1.5: this library is no longer supported
Couldn't find any versions for "@hint/hint-highest-available-document-mode" that matches "^5.0.14"
? Please choose a version of "@hint/hint-highest-available-document-mode" from this list: (Use arrow key
s)
❯ 5.0.13 
  5.0.12 
  5.0.11 
  5.0.10 
  5.0.9 
  5.0.8 
  5.0.7 
  5.0.6 
  5.0.5 
  5.0.4 
  5.0.3 
  5.0.2 
  5.0.1 
  5.0.0 
  4.2.3 
  4.2.2 
  4.2.1 
  4.2.0 
  4.1.11 
  4.1.10 
  4.1.9 
  4.1.8 
  4.1.7 
  4.1.6 
  4.1.5 
  4.1.4 
  4.1.3 
  4.1.2 
  4.1.1 
  4.1.0 
  4.0.2 
  4.0.1 
Couldn't find any versions for "@hint/hint-highest-available-document-mode" that matches "^5.0.14"
? Please choose a version of "@hint/hint-highest-available-document-mode" from this list: (Use arrow key
s)
❯ 5.0.13 
  5.0.12 
  5.0.11 
  5.0.10 
  5.0.9 
  5.0.8 
  5.0.7 
  5.0.6 
  5.0.5 
  5.0.4 
  5.0.3 
  5.0.2 
  5.0.1 
  5.0.0 
  4.2.3 
  4.2.2 
  4.2.1 
  4.2.0 
  4.1.11 
  4.1.10 
  4.1.9 
  4.1.8 
  4.1.7 
  4.1.6 
  4.1.5 
  4.1.4 
  4.1.3 
  4.1.2 
  4.1.1 
  4.1.0 
  4.0.2 
  4.0.1 
Couldn't find any versions for "@hint/hint-http-cache" that matches "^4.0.14"
? Please choose a version of "@hint/hint-http-cache" from this list: (Use arrow keys)
❯ 4.0.13 
  4.0.12 
  4.0.11 
  4.0.10 
  4.0.9 
  4.0.8 
  4.0.7 
  4.0.6 
  4.0.5 
  4.0.4 
  4.0.3 
  4.0.2 
  4.0.1 
  4.0.0 
  3.3.3 
  3.3.2 
  3.3.1 
  3.3.0 
  3.2.11 
  3.2.10 
  3.2.9 
  3.2.8 
  3.2.7 
  3.2.6 
  3.2.5 
  3.2.4 
  3.2.3 
  3.2.2 
  3.2.1 
  3.2.0 
  3.1.2 
  3.1.1 
(Move up and down to reveal more choices)(node:16742) MaxListenersExceededWarning: Possible EventEmitter memory leak detected. 11 keypress listeners added to [ReadStream]. Use emitter.setMaxListeners() to increase limit
(Use `node --trace-warnings ...` to show where the warning was created)
Couldn't find any versions for "@hint/hint-http-compression" that matches "^5.2.18"
? Please choose a version of "@hint/hint-http-compression" from this list: (Use arrow keys)
❯ 5.2.17 
  5.2.16 
  5.2.15 
  5.2.14 
  5.2.13 
  5.2.12 
  5.2.11 
  5.2.10 
  5.2.9 
  5.2.8 
  5.2.7 
  5.2.6 
  5.2.5 
  5.2.4 
  5.2.3 
  5.2.2 
  5.2.1 
  5.2.0 
  5.1.11 
  5.1.10 
  5.1.9 
  5.1.8 
  5.1.7 
  5.1.6 
  5.1.5 
  5.1.4 
  5.1.3 
  5.1.2 
  5.1.1 
  5.1.0 
  5.0.2 
  5.0.1 
Couldn't find any versions for "@hint/hint-content-type" that matches "^4.2.18"
? Please choose a version of "@hint/hint-content-type" from this list: (Use arrow keys)
❯ 4.2.17 
  4.2.16 
  4.2.15 
  4.2.14 
  4.2.13 
  4.2.12 
  4.2.11 
  4.2.10 
  4.2.9 
  4.2.8 
  4.2.7 
  4.2.6 
  4.2.5 
  4.2.4 
  4.2.3 
  4.2.2 
  4.2.1 
  4.2.0 
  4.1.11 
  4.1.10 
  4.1.9 
  4.1.8 
  4.1.7 
  4.1.6 
  4.1.5 
  4.1.4 
  4.1.3 
  4.1.2 
  4.1.1 
  4.1.0 
  4.0.2 
  4.0.1 
Couldn't find any versions for "@hint/hint-no-html-only-headers" that matches "^3.0.14"
? Please choose a version of "@hint/hint-no-html-only-headers" from this list: (Use arrow keys)
❯ 3.0.13 
  3.0.12 
  3.0.11 
  3.0.10 
  3.0.9 
  3.0.8 
  3.0.7 
  3.0.6 
  3.0.5 
  3.0.4 
  3.0.3 
  3.0.2 
  3.0.1 
  3.0.0 
  2.4.3 
  2.4.2 
  2.4.1 
  2.4.0 
  2.3.11 
  2.3.10 
  2.3.9 
  2.3.8 
  2.3.7 
  2.3.6 
  2.3.5 
  2.3.4 
  2.3.3 
  2.3.2 
  2.3.1 
  2.3.0 
  2.2.2 
  2.2.1 
Couldn't find any versions for "@hint/hint-strict-transport-security" that matches "^3.0.14"
? Please choose a version of "@hint/hint-strict-transport-security" from this list: (Use arrow keys)
❯ 3.0.13 
  3.0.12 
  3.0.11 
  3.0.10 
  3.0.9 
  3.0.8 
  3.0.7 
  3.0.6 
  3.0.5 
  3.0.4 
  3.0.3 
  3.0.2 
  3.0.1 
  3.0.0 
  2.4.3 
  2.4.2 
  2.4.1 
  2.4.0 
  2.3.11 
  2.3.10 
  2.3.9 
  2.3.8 
  2.3.7 
  2.3.6 
  2.3.5 
  2.3.4 
  2.3.3 
  2.3.2 
  2.3.1 
  2.3.0 
  2.2.2 
  2.2.1 
Couldn't find any versions for "@hint/hint-x-content-type-options" that matches "^4.0.14"
? Please choose a version of "@hint/hint-x-content-type-options" from this list: (Use arrow keys)
❯ 4.0.13 
  4.0.12 
  4.0.11 
  4.0.10 
  4.0.9 
  4.0.8 
  4.0.7 
  4.0.6 
  4.0.5 
  4.0.4 
  4.0.3 
  4.0.2 
  4.0.1 
  4.0.0 
  3.3.3 
  3.3.2 
  3.3.1 
  3.3.0 
  3.2.11 
  3.2.10 
  3.2.9 
  3.2.8 
  3.2.7 
  3.2.6 
  3.2.5 
  3.2.4 
  3.2.3 
  3.2.2 
  3.2.1 
  3.2.0 
  3.1.2 
  3.1.1 
warning hint > @hint/configuration-web-recommended > @hint/connector-puppeteer > @hint/utils-connector-tools > request@2.88.2: request has been deprecated, see https://github.com/request/request/issues/3142
```

.hintrc file:
```json
{
	"connector": {
		"name": "local"
	},
	"formatters": ["codeframe"],
	"extends": ["development", "progressive-web-apps", "accessibility"],
	"hints": {
		"axe/language": "off",
		"axe/text-alternatives": "off",
		"minified-js": "error",
		"meta-theme-color": "error",
		"button-type": "error",
		"create-element-svg": "error",
		"css-prefix-order": "error",
		"leading-dot-classlist": "error",
		"no-inline-styles": "error",
		"scoped-svg-styles": "error",
		"highest-available-document-mode": "error",
		"meta-charset-utf-8": "error",
		"meta-viewport": "error",
		"no-bom": "error",
		"compat-api/css": "error",
		"compat-api/html": [
			"error",
			{
				"ignore": ["meta[name=color-scheme]", "loading"]
			}
		],
		"disown-opener": "error",
		"no-protocol-relative-urls": "error",
		"sri": "error",
		"babel-config/is-valid": "error",
		"typescript-config/is-valid": "error",
		"typescript-config/target": "error",
		"webpack-config/is-installed": "error",
		"webpack-config/is-valid": "error",
		"webpack-config/module-esnext-typescript": "error",
		"webpack-config/no-devtool-in-prod": "error"
	},
	"parsers": ["webpack-config", "typescript-config"]
}
```

**Comment by ilan-dahan** [comment]
> Thanks for reporting this issue! Let me ping @vidorteg and @antross for investigation.

Sure, please know I'm available for further details as it's a hot concern for our team 👍

**Comment by ilan-dahan** [comment]
> Hi @ilan-dahan, thanks for reporting this issue. We identified the problem and we are working on a solution, ideally we can release a fix soon. I'll update this thread with any additional information

Great team! Thanks for your promptness and your global work, we do appreciate 👍

**Comment by vidorteg** [comment]
Hi @ilan-dahan, 

We just pushed a fixed version of webhint(version 7.0.1), it should contain the right dependency tree. Please let us know if this fixed the issue for issue and again thanks for reporting it :-D

**Comment by ilan-dahan** [comment]
> Hi @ilan-dahan, 
> 
> 
> 
> We just pushed a fixed version of webhint(version 7.0.1), it should contain the right dependency tree. Please feel free to reactivate if it still does not work and again thanks for reporting it :-D

Hi @vidorteg, it did solve the issue, you made it 😉!
Thank you for the quick patch!
